src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double lowerBase;
+    private double upperBase;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double lowerBase, double upperBase, double height) {
+        super(color, FigureList.ISOSCELES_TRAPEZOID.name());
+        this.lowerBase = lowerBase;
+        this.upperBase = upperBase;
+        this.height = height;
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead of passing the `FigureList.ISOSCELES_TRAPEZOID.name()` directly into the super constructor, store it in a variable and then pass the variable. This will improve the code readability.